generators/environment/templates/bin/pre-install.sh

```diff
@@ -11,7 +11,7 @@
 ##
 
 if [ ! -f src/sites/default/default.settings.php ]; then
-  echo "Please run pre-install after successfully building the docroot."
+  echo "There was an error building the docroot. Please run pre-install after it has been successfully built."
```

Comment:
There was not necessarily an error building the docroot. For CI, the main way the site is build (currently) is with the start.sh script. For local development, the start script is rarely used. If it is not rarely used, I consider that a very bad bug with the overall system, and not something to codify further. The main way to install the site is to run `rig project install` or `grunt install`, wherein a build may not have been done.